active_bots/telegrambot.py

```diff
@@ -32,6 +32,12 @@ def crawl(self, user):
                     logger.error("Unknown Telegram error code: " + str(update))
                 return reports
             user.save_seen_tg(update.update_id)
+            if update.message.photo:
+                tb.send_message(
+                    update.message.sender.id,
+                    "Sending Photos is not supported for privacy reasons. Can "
+                    "you describe it in a report instead?")
```

Comment:
```suggestion
                    "you describe it as text instead?")
```
It's not obvious that a report refers to writing a text message.